client/src/app/components/employee-settings/employee-settings.component.ts

```diff
@@ -70,15 +83,32 @@ export class EmployeeSettingsComponent implements OnInit {
   submit(): void {
     this.employee.is_active = this.isActive === 'active';
 
-    this.roles.forEach(role => {
+    this.roles.forEach((role) => {
       if (role.name === this.role) {
         this.employee.role = role.abbrev;
       }
     });
 
-    this.manageMembersService.updateClientInfo(this.employee, this.id).subscribe(response => {
-      location.reload();
-    });
+    this.manageMembersService
+      .updateClientInfo(this.employee, this.id)
+      .subscribe((response) => {
+        location.reload();
+      });
   }
 
+  submitPassword(): void {
+    this.employee.is_active = this.isActive === 'active';
```

Comment:
why do we need this here, if we're not submitting it?

Reply:
I deleted it.